verify.go

```diff
@@ -27,14 +28,16 @@ var (
 		RequiredMetadata:     []Metadata{MetaCreated, MetaKeyID},
 		DisallowedMetadata:   []Metadata{MetaAlgorithm}, // The algorithm should be looked up from the keyid not an explicit setting.
 		CreatedValidDuration: time.Minute * 5,           // Signatures must have been created within within the last 5 minutes
-		DateFieldSkew:        time.Minute,               // If the created parameter is present, the Date header cannot be more than a minute off.
```

Comment:
I got rid of DateFieldSkew (it was never actually used for enforcement) as this is not part of the spec and is related to an http header and not a signature component.